WooCommerce/Classes/ViewRelated/Orders/Order Details/Issue Refunds/Cells/RefundProductsTotalViewModel.swift

```diff
@@ -0,0 +1,9 @@
+import Foundation
+
+/// Represents products cost details for an order to be refunded. Meant to be rendered by `RefundProductsTotalTableViewCell`
+///
+struct RefundProductsTotalViewModel {
+    let productsTax: String
+    let productsSubtotal: String
+    let productsTotal: String
```

Comment:
nit: "products" are pretty much implied in the view model name, maybe the properties can be simplified? up to you!

```suggestion
    let tax: String
    let subtotal: String
    let total: String
```

Reply:
I'll leave it as it is to be aligned with the other cell's view models